### ISSUE TITLE
[ Dead Source ] BlogTruyen 

### ISSUE THREAD

**Issue by tuanbmt4152** [keiyoushi/tachiyomi-extensions-source#5452]
### Source name

BlogTruyen

### Source language

Vietnamese - Vi

### Source link

BlogTruyen.vn

### Other details

The site has officially closed.
![Screenshot_1](https://github.com/user-attachments/assets/f244068e-30f0-4493-8e87-0f737c59acaa)

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source website is down.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

**Comment by MikeZeDev** [comment]
Website redirect to Bumcheo, may want to update extension or make new one 🤷

**Comment by tuanbmt4152** [comment]
> Website redirect to Bumcheo, may want to update extension or make new one 🤷

The site is now redirected to writing short stories and creating memes.

**Comment by MikeZeDev** [comment]
> > Website redirect to Bumcheo, may want to update extension or make new one 🤷
> 
> The site is now redirected to writing short stories and creating memes.

I just got blocked by cloudflare despites having VT ip so i cant say more

**Comment by dragonx943** [comment]
> ### Source name
> BlogTruyen
> 
> ### Source language
> Vietnamese - Vi
> 
> ### Source link
> BlogTruyen.vn
> 
> ### Other details
> The site has officially closed. ![Screenshot_1](https://private-user-images.githubusercontent.com/71616861/374988011-f244068e-30f0-4493-8e87-0f737c59acaa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkwMTIwMTUsIm5iZiI6MTcyOTAxMTcxNSwicGF0aCI6Ii83MTYxNjg2MS8zNzQ5ODgwMTEtZjI0NDA2OGUtMzBmMC00NDkzLThlODctMGY3MzdjNTlhY2FhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE1VDE3MDE1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTllOWIzNDQ4NWY1ODE0NDA1MzYyNDYyOTJmOWM4MTUxOTdjYjcwZmUwZGI5N2M2MjRhMGJkMTk0YTQ4YTdjZDcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.pUV-XeV9RUXdgORe8ImncVN7Lm4wYI2Ngnj-H1mH3JE)
> 
> ### Acknowledgements
> * [x]  I have updated all installed extensions.
> * [x]  I have opened WebView and checked that the source website is down.
> * [x]  I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
> * [x]  I have written a meaningful title with the source name.
> * [x]  I will fill out all of the requested information in this form.

@tuanbmt4152 @MikeZeDev fixed all, please wait for next kotatsu version !!!

![image](https://github.com/user-attachments/assets/1684242a-6729-4504-8221-3cd42cc92dbe)
